src/core/Makefile.am

```diff
@@ -184,7 +195,8 @@ noinst_HEADERS                      = \
     thread/thread_tlvs.hpp            \
     thread/thread_uris.hpp            \
     thread/topology.hpp               \
-    utils/global_address.hpp          \
+    thread/global_address.hpp         \
+    utils/iid_generator.hpp           \
     $(NULL)
```

Comment:
Please add all these files, including the `dhcp_client.cpp` and `dhcp_server.cpp` to:

```
etc/visual-studio/libopenthread.vcxproj
etc/visual-studio/libopenthread.vcxproj.filters
etc/visual-studio/libopenthread_k.vcxproj
etc/visual-studio/libopenthread_k.vcxproj.filters
```